mazegen.py

```diff
@@ -0,0 +1,4 @@
+import maze
+
+mz = maze.MazeBuilder(10,20)
+print(mz)
```

Comment:
No new line at the end of file
```suggestion
print(mz)

```